src/librustc_codegen_llvm/llvm/ffi.rs

```diff
@@ -1002,6 +1002,36 @@ extern "C" {
                          RHS: &'a Value,
                          Name: *const c_char)
                          -> &'a Value;
+    pub fn LLVMBuildNSWAdd(B: &Builder<'a>,
+                         LHS: &'a Value,
+                         RHS: &'a Value,
+                         Name: *const c_char)
+                         -> &'a Value;
```

Comment:
Nit: the arguments are misaligned - you could either align them or indent them like we do nowadays, i.e.:
```rust
    pub fn LLVMBuildNSWAdd(
        B: &Builder<'a>,
        LHS: &'a Value,
        RHS: &'a Value,
        Name: *const c_char,
    ) -> &'a Value;
```